examples/samples/KOSMAPOSL_mhkem.par

```diff
@@ -0,0 +1,107 @@
+KOSMAPOSLParameters :=
```

Comment:
additional template file I guess. Please say at the start how it differs from the previous one. The name doesn't quite tell me...